README.md

```diff
@@ -16,7 +16,7 @@ The `-c` option takes a map with the POA contracts' addresses in JSON format. Yo
 current maps for the main and test network the `contracts` folder. By default, it uses `core.json`,
 for the main network.
 
-The `-p` option takes a time interval in hours, days, months, etc. E.g. `-p "10 weeks"` will only count participation in ballots that were created within the last 10 weeks.
+The `-p` option takes a time interval in hours, days, months, etc. E.g. `-p "10 weeks"` will only count participation in ballots that were created within the last 10 weeks. Alternatively, instead of a _time_, you can specify the earliest block _number_ with the `-b` option.
```

Comment:
I think that explicitly saying that the block number should passed in as a decimal integer (as opposed to the hex encoded block number format that the JSON-RPC uses), may be helpful here.

Reply:
Good point, I added that.